### DISCUSSION TITLE
How to encode/decode for Adobe RGB/Display P3?

### DISCUSSION THREAD

**Discussion by hminle** [colour-science/colour#755] [Q&A] [answered]
Hi Colour Team,

How could I apply decode/encode (de-gamma/encode gamma) for AdobeRGB and DisplayP3?
I see in the library, you guys have: `colour.models.cctf_encoding_ProPhotoRGB` and `colour.models.cctf_decoding_ProPhotoRGB` for ProPhoto RGB.
But I cannot find those functions for AdobeRGB/DisplayP3.

Thanks,
Hoang

**Comment by sobotka** [comment, marked as answer]
https://colour.readthedocs.io/en/latest/generated/colour.models.ADOBE_RGB_1998_COLOURSPACE.html?highlight=Adobe

https://colour.readthedocs.io/en/latest/generated/colour.models.P3_D65_COLOURSPACE.html?highlight=Display%20%20P3

https://colour.readthedocs.io/en/latest/generated/colour.models.PROPHOTO_RGB_COLOURSPACE.html#colour.models.PROPHOTO_RGB_COLOURSPACE

You’ll want to use the CCTF functions.

https://colour.readthedocs.io/en/latest/colour.models.html#rgb-colourspaces

**Comment by hminle** [comment]
Thank @sobotka :)

**Comment by sobotka** [comment]
Let me know if you need help!
